include/torch-mlir/Dialect/Torch/IR/TorchOps.td

```diff
@@ -1326,4 +1326,113 @@ def Torch_DtypeCalculateYieldDtypesOp : Torch_Op<"dtype.calculate.yield.dtypes",
   let hasVerifier = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// Torch c10d Functional Communication Ops
+//===----------------------------------------------------------------------===//
+// These ops is manully added because nightly-build torch's signature is
+// not convergent. Generate them if torch has stable op signature.
```

Comment:
nit: Can you also mention something like "Autogenerated by `./build_tools/update_torch_ods.sh` with torch==(some version)`", so that people don't manually modify these